lib/sycamore/sycamore/query/result.py

```diff
@@ -0,0 +1,45 @@
+import io
+from typing import Any, Dict, Optional
+
+from pydantic import BaseModel
+
+from sycamore.query.logical_plan import LogicalPlan
+from sycamore import DocSet
+
+
+class SycamoreQueryResult(BaseModel):
```

Comment:
🙇

Reply:
I cannot tell if this is someone bowing in thanks for smacking their head on the table

Reply:
Bowing in thanks!